### DISCUSSION TITLE
Need help in using customldif files

### DISCUSSION THREAD

**Discussion by MuraliBabuChinni** [jp-gouin/helm-openldap#183] [Q&A]
I am trying to give customLdif files in values.yaml. But after installing helm chart I don't see my customldif files in ldapsearch command. 
In the ldapsearch command output, I am seeing only default ldif files.

Do I need to make changes any where else other than values.yaml?

Please HELP!!

**Comment by jp-gouin** [comment]
Hey , there are some working examples available [here](https://github.com/jp-gouin/helm-openldap/blob/master/advanced_examples/MemberOf.md) 

also when installing and reinstalling the chart to test changes in `customLdif ` you have to wipe out any `pvc` related to openldap

**Comment by MuraliBabuChinni** [comment]
Hi @jp-gouin ,
Wipe out any pvc and pv helped Thanks :)
one more query 
can I use hdb olcdatabase instead of mdb
example:
dn: olcDatabase={1}hdb,cn=config
changetype: modify
delete: olcAccess

and also I see this error message
_6698c205.01d0180e 0x7f2f2afe86c0 conn=1013 op=1 MOD attr=olcModuleLoad
6698c205.01da9f01 0x7f2f2afe86c0 lt_dlopenext failed: (ppolicy) file not found
6698c205.01dafbc1 0x7f2f2afe86c0 olcModuleLoad: value #0: <olcModuleLoad> handler exited with 1!
6698c205.01db8153 0x7f2f2afe86c0 conn=1013 op=1 RESULT tag=103 err=80 qtime=0.000009 etime=0.000808 text=<olcModuleLoad> handler exited with 1
ldap_modify: Other (e.g., implementation specific) error (80)_

when trying to load ppolicy module
10-ppolicy-module.ldif: |-
    dn: cn=module{0},cn=config
    changeType: modify
    add: olcModuleLoad
    olcModuleLoad: ppolicy
